docs/LocalDevelopmentGuide.rst

```diff
@@ -127,6 +127,9 @@ ngrok gives you. Note that it is very important to use the HTTPS URL,
 Localtunnel
 ~~~~~~~~~~~
 
+    Note: We no longer recommend localtunnel because of reliability issues.
+    Tread with caution.
```

Comment:
Super unfortunate that localtunnel doesn't perform well, but it sure beats changing my URLs when using ngrok imo - can we update the disclaimer to just say that localtunnel has reliability issues, instead of changing the recommendation? I would still recommend it for folks who do sporadic contributions, because of the ability to keep your URL